tests/qm-sanity-test/test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+source ../e2e/lib/utils
+
+# Verify qm is up and running
+check_qm_is_up(){
+    if [ "$(systemctl is-active qm)" == "active" ]; then
+        info_message "check_qm_is_up(): qm is active"
+    else
+        info_message "check_qm_is_up(): qm is not active"
+        exit 1
+    fi
+}
+
+# Verify bluechi nodes are connected
+check_bluechi_nodes_connected(){
+    expected_result="Connected to controller as 'qm.localrootfs'"
+    if [[ "$(podman exec -it qm bash -c "systemctl status bluechi-agent" \
+    | tail -2)" =~ ${expected_result} ]]; then
+        info_message "check_bluechi_nodes_connected: \
+        QM bluechi-agent is connected to controller"
+    else
+        info_message "check_bluechi_nodes_connected: \
+        QM bluechi-agent is not connected to controller"
+        exit 1
+    fi
+
+    expected_result="Connected to controller as 'localrootfs'"
+    if [[ "$(systemctl status bluechi-agent | tail -2)" =~ ${expected_result} ]]; then
+        info_message "check_bluechi_nodes_connected: \
+        ASIL bluechi-agent is connected to controller"
+    else
+        info_message "check_bluechi_nodes_connected: \
+        ASIL bluechi-agent is not connected to controller"
+        exit 1
+    fi
+}
+
+# Verify podman run and exec container inside qm with service file
+check_podman_run_and_exec(){
+    info_message "check_podman_run_and_exec(): \
+    prepare quadlet files for qm-sanity-test.container"
+    cat > "/etc/qm/containers/systemd/qm-sanity-test.container" <<EOF
+[Unit]
+Description=The sleep container
+After=local-fs.target
+
+[Container]
+Image=registry.access.redhat.com/ubi9-minimal:latest
+Exec=sleep 1000
+
+[Install]
+# Start by default on boot
+WantedBy=multi-user.target default.target
+EOF
+
+    info_message "check_podman_run_and_exec(): qm-sanity-test reload & restart"
+    exec_cmd "podman exec qm systemctl daemon-reload"
+    exec_cmd "podman exec qm systemctl start qm-sanity-test"
+    exec_cmd "podman exec qm systemctl status qm-sanity-test | grep -i started"
+
+    exec_cmd "podman exec qm podman run alpine echo hello"
+}
+
+# Verify /var partition exist
+check_var_partition_exist(){
+    if [[ "$(podman exec -it qm bash -c "ls /var")" =~ "No such file or directory" ]]; then
```

Comment:
I would use from the host lsblk or fdisk to identify partition for /var /
fdisk -l -o Device,Size,Type

verify host /var is on different partition
Difference between c9s and ostree in out put exist

Reply:
What does the output of `lsblk` in ostree look like? 
I tried with a ostree image "auto-osbuild-qemu-rhivos9-minimal-ostree-x86_64-8657563.d519d68e.qcow2”, but qm is inactive on this image. So I don't know if the output in the following image is correct:
![image](https://github.com/containers/qm/assets/54343860/29b73169-e0fd-451e-b046-ad661a12518a)

Reply:
here, you can see that there is not separate partition for /var
see this 
```
[root@8e2b5d13-97aa-458d-a321-7533436e4e2c ~]# lsblk
NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
loop0         7:0    0  12.7M  1 loop 
nvme0n1     259:0    0   250G  0 disk 
├─nvme0n1p1 259:1    0   100M  0 part /boot/efi
├─nvme0n1p2 259:2    0   300M  0 part /boot
├─nvme0n1p3 259:3    0   2.4G  0 part /var
└─nvme0n1p4 259:4    0 247.2G  0 part /sysroot/ostree/deploy/rhivos/var
                                      /etc
                                      /sysroot

```

Reply:
@Yarboa thank you for sharing.

Reply:
done